README.md

```diff
@@ -61,6 +61,7 @@ VERSIONS = [
 
 dependencies {
     implementation "io.quarkiverse.scala:quarkus-scala3:${VERSIONS.QUARKUS_SCALA3}"
+    implementation "io.quarkiverse.scala:quarkus-scala3-deployment:${VERSIONS.QUARKUS_SCALA3}"
```

Comment:
I don't think this is right. You shouldn't have to add the `-deployment` dependencies.

Reply:
done